__tests__/app.test.js

```diff
@@ -77,6 +77,59 @@ describe("GET /api/reviews", () => {
         expect(reviews).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("GET - status:200, can order array of reviews by ascending value", () => {
+    return request(app)
+      .get("/api/reviews?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSortedBy("created_at");
+      });
+  });
+  test("GET - status:400, invalid order query!", () => {
+    return request(app)
+      .get("/api/reviews?order=something bad")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid Order Query!");
+      });
+  });
+  test("GET - status:200, can sort array of reviews by specified sort_by value", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=title")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+          expect(reviews).toBeSortedBy("title", { descending: true });;
+      });
+  });
+  test("GET - status:400, invalid sort query!", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=something bad")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid Sort Query!");
+      });
+  });
+  test("GET - status:200, can filter reviews by category", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review.category).toBe("social deduction");
+        });
+      });
+  });

```

Comment:
What happens with an invalid category? What about a valid category with no associated reviews? Looks like you've considered it in your model, but not tested it.